tests/pkg_littlefs2/Makefile

```diff
@@ -0,0 +1,32 @@
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove \
+                             arduino-leonardo \
+                             arduino-nano \
+                             arduino-uno \
+                             i-nucleo-lrwan1 \
+                             nucleo-f031k6 \
+                             nucleo-f042k6 \
+                             nucleo-l031k6 \
+                             nucleo-l053r8 \
+                             nucleo-f030r8 \
+                             stm32f0discovery \
+                             stm32l0538-disco \
+
+BOARD_BLACKLIST := chronos \
+                   msb-430 \
+                   msb-430h \
+                   telosb \
+                   wsn430-v1_3b \
+                   wsn430-v1_4 \
+                   z1 \
```

Comment:
predates `FEATURES_BLACKLIST += arch_msp430`.

(Is this even still needed after the msp430 toolchain update?)

Reply:
I'll ask Murdock

Reply:
> (Is this even still needed after the msp430 toolchain update?)

Still required, both issues in missing intrinsics used by LittleFS(v2) and in addition there are also issues with the word width on msp430 and bit shifts used by LittleFS.

Blacklist in the package `Makefile.dep` it is